BUILD

```diff
@@ -23,3 +24,49 @@ buildozer_binary(
     name = "buildozer",
     visibility = ["//visibility:public"],
 )
+
+bzl_library(
+    name = "defs",
+    srcs = ["defs.bzl"],
+    visibility = ["//visibility:public"],
+)
+
+bzl_library(
+    name = "rules",
+    srcs = ["rules.bzl"],
+    deps = [
+        "@bazel_skylib//lib:shell",
+    ],
+)
+
+# MARK: - Integration Test
+
+bzl_library(
+    name = "bazel_versions",
+    srcs = ["bazel_versions.bzl"],
+)
```

Comment:
Besides the build_test no-op use case is there a reason  to have these ?

Reply:
Just for the `build_test`. If we remove that, I think that these can go away. Actually, your comment reminded me that I did not add all of these targets to the `build_test`. 😳